src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -16,9 +16,36 @@
 import org.kohsuke.args4j.CmdLineException;
 import org.kohsuke.args4j.CmdLineParser;
 import org.kohsuke.args4j.Option;
-import org.kohsuke.args4j.spi.StringArrayOptionHandler;
+import org.kohsuke.args4j.OptionDef;
+import org.kohsuke.args4j.spi.OptionHandler;
+import org.kohsuke.args4j.spi.Parameters;
+import org.kohsuke.args4j.spi.Setter;
 
 public class Options {
+  public static class StringArrayOptionHandler extends OptionHandler<String> {
```

Comment:
doesn't need to be public, does it?

Reply:
I think it does, though I'll double-check

Reply:
package visibility (no keyword) should be enough if you only need it from the test.

Reply:
It also needs to be discoverable by args4j, which uses reflection to find the class. I had run into an issue with the visibility, but I'll see if args4j can find it with package visibility later today.

Reply:
Looking at it now it appears, yes, this class does need to be `public`. When it is package private I get the following from args4j:

```text
Exception in thread "main" org.kohsuke.args4j.IllegalAnnotationError: java.lang.IllegalAccessException: Class org.kohsuke.args4j.OptionHandlerRegistry$DefaultConstructorHandlerFactory can not access a member of class com.google.javascript.clutz.Options$StringArrayOptionHandler with modifiers "public"
	at org.kohsuke.args4j.OptionHandlerRegistry$DefaultConstructorHandlerFactory.getHandler(OptionHandlerRegistry.java:226)
	at org.kohsuke.args4j.OptionHandlerRegistry.createOptionHandler(OptionHandlerRegistry.java:188)
	at org.kohsuke.args4j.CmdLineParser.createOptionHandler(CmdLineParser.java:178)
	at org.kohsuke.args4j.CmdLineParser.addOption(CmdLineParser.java:146)
	at org.kohsuke.args4j.ClassParser.parse(ClassParser.java:34)
	at org.kohsuke.args4j.CmdLineParser.<init>(CmdLineParser.java:96)
	at org.kohsuke.args4j.CmdLineParser.<init>(CmdLineParser.java:71)
	at com.google.javascript.clutz.Options.<init>(Options.java:164)
	at com.google.javascript.clutz.DeclarationGenerator.main(DeclarationGenerator.java:239)
Caused by: java.lang.IllegalAccessException: Class org.kohsuke.args4j.OptionHandlerRegistry$DefaultConstructorHandlerFactory can not access a member of class com.google.javascript.clutz.Options$StringArrayOptionHandler with modifiers "public"
	at sun.reflect.Reflection.ensureMemberAccess(Reflection.java:102)
	at java.lang.reflect.AccessibleObject.slowCheckMemberAccess(AccessibleObject.java:296)
	at java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:288)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:413)
	at org.kohsuke.args4j.OptionHandlerRegistry$DefaultConstructorHandlerFactory.getHandler(OptionHandlerRegistry.java:222)
	... 8 more
```